app/views/payments/new.html.erb

```diff
@@ -0,0 +1,19 @@
+
+<div class="category-header">
+     <a href="<%= categories_path %>">
+            <i class="fa-solid fa-arrow-left"></i>
+        </a>
+    <h2>New Transaction</h2>
+    <a href="/logout">
+        <i class="fa-solid fa-right-from-bracket"></i>
+    </a>
+</div>
+
+<%= form_with(model: @payment, url: category_payments_path) do |form| %>
+   
+<div class="form-container">
+        <%= form.text_field :name, placeholder: "Name" %>
+        <%= form.text_field :amount, placeholder: "Amount" %>
+         <%= form.submit "Create", class: "submit-button" %>
+    </div>
```

Comment:
- [ ] Kindly add another field for category selection (check box or dropdown menu, etc.), this is part of the project requirements.

![selection](https://github.com/atok624/Budget-App/assets/53124548/517a4c5d-bf5d-4034-bace-4f811590734c)

Reply:
Resoved